internal/routes.go

```diff
@@ -74,7 +74,7 @@ func fetchMedia(c *gin.Context) (chan Movie, chan TV, chan Person) {
 				<-guardCh
 			}(id, personCh)
 		default:
-			log.Fatal("This is not the movie type you seek")
+			log.Fatal("This is not the movie type you seek - skipping to next media")
```

Comment:
This wont skip, it's just break and die :D Fatal consists of an `os.Exit(1)` https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/log/log.go;l=363

It needs the weird `Id: -1`-hack if we want it to skip

Reply:
```suggestion
			log.Println("This is not the movie type you seek - skipping to next media")
```